src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -218,12 +218,11 @@ class HoldRequest extends React.Component {
         <div className="nypl-request-page-header">
           <div className="nypl-full-width-wrapper">
             <div className="row">
-              <div className="nypl-column-three-quarters">
+              <div className="nypl-column-full">
                 <Breadcrumbs
-                  query={searchKeywords}
+                  query={`q=${searchKeywords}`}
```

Comment:
Yeah, I didn't want to add too much logic inside the component and that string need to be updated anyways (to include filters if available).